ui/src/containers/ExportDownloadManager/index.js

```diff
@@ -17,18 +19,34 @@ class ExportDownloadManager extends Component {
   render = () => {
     const { models, isLoading } = this.props;
     if (isLoading) return (<Spinner />);
-
     return (
       <div>
-        { models.map((model, id) =>
-          <DownloadListItem key={id} model={model} deleteModel={this.props.deleteModel} />
+        { 
+          models.map((model, id) =>
+          <DownloadListItem
+            key={id}
+            model={model}
+            exporationExport={this.props.organisation.getIn(['settings', 'EXPIRE_EXPORTS'])}
```

Comment:
Correct spelling to `expirationExport` or maybe rename this to `exportExpirationSettings` so it's a bit more obvious what it contains in the child component.